default/stringtables/en.txt

```diff
@@ -6572,6 +6572,12 @@ SITREP_TECH_RESEARCHED
 SITREP_TECH_RESEARCHED_LABEL
 Tech Researched
 
+SITREP_EMPIRE_TECH_RESEARCHED_DETECTION
+The %empire% upgraded their detection technology to %rawtext:dstrength% [[encyclopedia DETECTION_TITLE]]
```

Comment:
Rephrase more passive... The %empire% detection strength has upgraded to... or somesuch.